x-pack/test/functional/page_objects/infra_home_page.ts

```diff
@@ -16,6 +16,7 @@ export function InfraHomePageProvider({ getService }: FtrProviderContext) {
 
   return {
     async goToTime(time: string) {
+      await testSubjects.missingOrFail('loadingMessage', { timeout: 20000 });
```

Comment:
I don't understand the reasoning for putting this here.  Wouldn't it cause goToTime to fail if the loadingMessage is found?
In general, I think what ever action causes the loading should wait for the loadingMesage to clear (post action wait), instead of checking that it's not there before doing the next action (pre action check).

Reply:
@LeeDr I checked locally and that loading is happening after url navigation, I added `await pageObjects.infraHome.waitForLoading();` in before method